RubeesEat.UnitTests/Model/PersonTest.cs

```diff
@@ -0,0 +1,34 @@
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+[TestFixture]
+public class PersonTest
+{
+    [Test]
+    public void Person_Constructor_AssignsPropertiesCorrectly()
+    {
+        Guid expectedGuid = Guid.NewGuid();
+        string expectedFirstName = "test";
+        string expectedLastName = "test";
+        var person = new Person(expectedGuid, expectedFirstName, expectedLastName);
+        Assert.That(person.Id, Is.EqualTo(expectedGuid));
+        Assert.That(person.FirstName, Is.EqualTo(expectedFirstName));
+        Assert.That(person.LastName, Is.EqualTo(expectedLastName));
+
+    }
+    
+    [Test]
+    [TestCase("Test2")]
+    [TestCase("3Test")]
+    [TestCase("?Test")]
+    [TestCase("1Test1")]
+    [TestCase("Te st")]
+    [TestCase(".")]
+    [TestCase("")]
+    [TestCase(" ")]
+    public void Person_ConstructorWithInvalidName_ThrowsException(string name)
+    {
+        Assert.That(() => new Person(Guid.NewGuid(), name, name), Throws.TypeOf<ArgumentException>().With.Message.EqualTo("Name must only consist of letters."));
```

Comment:
When lines become too long it is better for readability to split them into multiple lines. I would suggest splitting up all parameters on new lines and then splitting up the `.With.Message...` thing on its own line like this:

```suggestion
        Assert.That(
            () => new Person(Guid.NewGuid(), name, name), 
            Throws.TypeOf<ArgumentException>()
                .With.Message.EqualTo("Name must only consist of letters."));
```